extension/page-detect.js

```diff
@@ -1,7 +1,5 @@
 window.pageDetect = (() => {
-	const isGistCheck = location.hostname === 'gist.github.com';
-
-	const isGist = () => isGistCheck;
+	const isGist = () => location.hostname === 'gist.github.com';
```

Comment:
Yeah, I changed it for testing purposes. We *technically* can test without changing it, but it requires deleting a reference from node's `require.cache` and re-requiring it each time we need to make a change to `location.hostname`, which doesn't seem like a great thing to be doing in tests if it can be avoided.